resources/js/frontend/wc-payplug-apple_pay_cart-blocks.js

```diff
@@ -42,18 +42,19 @@ const ApplePayCart = ( props ) =>{
 					'name',
 				],
 			}
+			request.requiredShippingContactFields = [
```

Comment:
this should only be mandatory for non-virtual products, if the order doesn't required a shipping this may not be filled